docs/Overview.md

```diff
@@ -73,6 +73,26 @@ The `statusReportingDelay` property allows clients to customize the reporting de
 |:--:|
 | _**Status reporting delay example**_ |
 
+### Sender status transition counters
+
+All sender specific domain statuses have an associated status transition counter property. These increment each time the associated status transitions to a less healthy state. Transitions to/from neutral states like `Inactive` or `NotUsed` are ignored.
+
+The intention is that these properties store historical negative trend transitions for each status.
+
+The list of all status transition counter properties is:
+
+* linkStatusTransitionCounter
+* transmissionStatusTransitionCounter
+* externalSynchronizationStatusTransitionCounter
+* essenceStatusTransitionCounter
+
+Devices MUST be able to reset ALL status transition counter properties in the following two ways:
+
+* When a sender activation occurs
+* When a client invokes the `ResetStatusTransitionCounters` method
```

Comment:
Same as the comment made on BCP-008-01 for this part of the spec: https://github.com/AMWA-TV/bcp-008-01/pull/18#discussion_r1963958631